### PR TITLE
Mt 1215

### PR DESCRIPTION
**Pull request by craigrouse**

### CODE REVIEW

File: Sources/FirebaseRemoteCommand.swift [hunk 1]
```diff
@@ -237,103 +185,156 @@ public class FirebaseRemoteCommand: RemoteCommand {
...
+        "param_ad_platform": AnalyticsParameterAdPlatform,
+        "param_ad_source": AnalyticsParameterAdSource,
+        "param_ad_unit_name": AnalyticsParameterAdUnitName,
+        "param_affiliation": AnalyticsParameterAffiliation,
+        "param_cp1": AnalyticsParameterCP1,
+        "param_campaign": AnalyticsParameterCampaign,
+        "param_campaign_id": AnalyticsParameterCampaignID, // Version 8.12.1
+        "param_character": AnalyticsParameterCharacter,
+//            "param_checkout_step": AnalyticsParameterCheckoutStep,
+//            "param_checkout_option": AnalyticsParameterCheckoutOption,
```
Comment (trister1997) [inline Sources/FirebaseRemoteCommand.swift]: Should we remove the commented ones? Or should they not be commented?

File: TealiumFirebaseExample/TealiumFirebaseExample/TealiumHelper.swift [hunk 2]
```diff
@@ -27,7 +27,8 @@ class TealiumHelper {
     var tealium: Tealium?
     
     // JSON Remote Command
-    let firebaseRemoteCommand = FirebaseRemoteCommand(type: .remote(url: "https://tags.tiqcdn.com/dle/tealiummobile/demo/firebase.json"))
+//    let firebaseRemoteCommand = FirebaseRemoteCommand(type: .remote(url: "https://tags.tiqcdn.com/dle/tealiummobile/demo/firebase.json"))
```
Comment (trister1997) [inline TealiumFirebaseExample/TealiumFirebaseExample/TealiumHelper.swift]: Remove the commented code

File: tealium-xcframeworks/TealiumFirebase.xcframework/ios-arm64_x86_64-simulator/TealiumFirebase.framework/Headers/FirebaseConstants.swift [hunk 3]
```diff
@@ -0,0 +1,51 @@
+//
```
Comment (trister1997) [inline tealium-xcframeworks/TealiumFirebase.xcframework/ios-arm64_x86_64-simulator/TealiumFirebase.framework/Headers/FirebaseConstants.swift]: Are these files always tracked? Should they be?

File: Sources/FirebaseRemoteCommand.swift (hunk 1 above)
Comment (Enricoza) [reply Sources/FirebaseRemoteCommand.swift]: should be removed as they are removed from the firebase APIs

File: tealium-xcframeworks/TealiumFirebase.xcframework/ios-arm64_x86_64-simulator/TealiumFirebase.framework/Headers/FirebaseConstants.swift (hunk 3 above)
Comment (Enricoza) [reply tealium-xcframeworks/TealiumFirebase.xcframework/ios-arm64_x86_64-simulator/TealiumFirebase.framework/Headers/FirebaseConstants.swift]: They were tracked by mistake

**Pull request merged**